doc/sphinx/source/recipes/recipe_carvalhais2014nat.rst

```diff
@@ -0,0 +1,342 @@
+
+
+Script land_carbon_cycle/diag_zonal_turnover.py
+...............................................
+
+  * Required settings:
+
+    * ``obs_variable``: {``str``} list of the variable(s) to be read from the
+      observation files
+
+  * Optional settings:
+
+    * ``ax_fs``: {``float``, 7.1} - fontsize in the figure.
+    * ``fill_value``: {``float``, nan} - fill value to be used in analysis and
+      plotting.
+    * ``valrange_x``: {``tuple``, (2, 1000)} - range of turnover values in the
+      X - axis.
+    * ``valrange_y``: {``tuple``, (-70, 90)} - range of latitudes in the Y -
+      axis.
+    * ``bandsize``: {``float``, 9.5} - size of the latitudinal rolling window
+      in degrees. One latitude row if set to ``None``.
+    * ``gpp_threshold``: {``float``, 0.01} - The threshold of gpp in
+      `kg m^{-2} yr^{-1}` below which the grid cells are masked.
+
+
+Script land_carbon_cycle/diag_zonal_correlation.py
+..................................................
+
+  * Required settings:
+
+    * ``obs_variable``: {``str``} list of the variable(s) to be read from the
+      observation files
+
+  * Optional settings:
+
+    * ``ax_fs``: {``float``, 7.1} - fontsize in the figure.
+    * ``fill_value``: {``float``, nan} - fill value to be used in analysis and
+      plotting.
+    * ``correlation_method``: {``str``, pearson | spearman} - correlation
+      method to be used while calculating the zonal correlation.
+    * ``min_points_frac: {``float``, 0.125} - minimum fraction of valid points
+      within the latitudinal band for calculation of correlation.
+    * ``valrange_x``: {``tuple``, (-1, 1)} - range of correlation values in the
+      X - axis.
+    * ``valrange_y``: {``tuple``, (-70, 90)} - range of latitudes in the Y -
+      axis.
+    * ``bandsize``: {``float``, 9.5} - size of the latitudinal rolling window
+      in degrees. One latitude row if set to ``None``.
+    * ``gpp_threshold``: {``float``, 0.01} - The threshold of gpp in
+      `kg m^{-2} yr^{-1}` below which the grid cells are masked.
+
+
+Required Variables
+------------------
+
+* *tas* (atmos, monthly, longitude, latitude, time)
+* *pr* (atmos, monthly, longitude, latitude, time)
+* *gpp* (land, monthly, longitude, latitude, time)
+* *cVeg* (land, monthly, longitude, latitude, time)
+* *cSoil* (land, monthly, longitude, latitude, time)
+
+.. _observations:
+
+Observations
+------------
+
+The observations needed in the diagnostics are publicly available for download
+from the .. _Data Portal of the Max Planck Institute for Biogeochemistry:
+http://www.bgc-jena.mpg.de/geodb/BGI/tau4ESMValTool.php after registration.
+
+Due to inherent dependence of the diagnostic on uncertainty estimates in
+observation, the data needed for each diagnostic script are processed at
+different spatial resolutions (as in Carvalhais et al., 2014), and provided in
+11 different resolutions (see Table 1).
+
+Table 1. A summary of the observation datasets at different resolutions.
```

Comment:
This is not possible because the uncertainties cannot be regridded to any resolution. The original full ensembles have to be accessed to generate all ensemble members from which the uncertainties are explicitly calculated. For zonal means, it's even more calculation steps and needs data for ctotal and gpp separately, which are not open. Som we decided to provide the preprocessed data.

Reply:
Since this question pops up by everyone who comes across this diagnostic for the first time, maybe it's worth explaining it in the documentation? A short note to that effect seems appropriate, no?

Reply:
I have added a few lines to the documentation to clarify.
"Note that the uncertainties were
estimated at the resolution of the selected models, and, thus, only the 
pre-processed observed data can be used with the recipe. 
It is not possible to use regridding functionalities of ESMValTool to regrid 
the observational data to other spatial resolutions, as the uncertainty 
estimates cannot be regridded."